addons/event/migrations/11.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,337 @@
+                                <a href="https://www.google.com/calendar/render?action=TEMPLATE&text=${object.event_id.name}&dates=${date_begin}/${date_end}&location=${location}" style="padding:3px 5px;border:1px solid #875A7B;color:#875A7B;text-decoration:none;border-radius:3px;" target="new"><img src="/web_editor/font_to_img/61525/rgb(135,90,123)/16" style="vertical-align:middle;" height="16"> Google</a>
+                                <a href="https://bay02.calendar.live.com/calendar/calendar.aspx?rru=addevent&summary=${object.event_id.name}&dtstart=${date_begin}&dtend=${date_end}&location=${location}" style="padding:3px 5px;border:1px solid #875A7B;color:#875A7B;text-decoration:none;border-radius:3px;" target="new"><img src="/web_editor/font_to_img/61525/rgb(135,90,123)/16" style="vertical-align:middle;" height="16"> Outlook</a>
+                                <a href="https://calendar.yahoo.com/?v=60&view=d&type=20&title=${object.event_id.name}&in_loc=${location}&st=${format_tz(object.event_id.date_begin, tz='UTC', format='%Y%m%dT%H%M%S')}&et=${format_tz(object.event_id.date_end, tz='UTC', format='%Y%m%dT%H%M%S')}" style="padding:3px 5px;border:1px solid #875A7B;color:#875A7B;text-decoration:none;border-radius:3px;" target="new"><img src="/web_editor/font_to_img/61525/rgb(135,90,123)/16" style="vertical-align:middle;" height="16"> Yahoo</a>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    % if object.event_id.address_id:
+    <div style="width:598px;margin:0px auto;border-left:1px solid #dddddd;border-right:1px solid #dddddd;border-bottom:1px solid #dddddd;">
+        <a href="https://maps.google.com/maps?q=${location}" target="new">
+            <img src="http://maps.googleapis.com/maps/api/staticmap?autoscale=1&size=598x200&maptype=roadmap&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xa5117d%7Clabel:%7C${location}" style="vertical-align:bottom;" />
+        </a>
+    </div>
+    % endif
+    <table style="width:600px;margin:0px auto;text-align:center;">
+        <tbody>
+            <tr>
+                <td style="padding-top:10px;font-size: 12px;">
+                    <p style="margin:0px 0px 9px 0px;padding-top:10px;">Sent by ${object.company_id.name}</p>
+                    % if 'website_url' in object.event_id and object.event_id.website_url:
+                    <div>
+                        Discover <a href="/event" style="text-decoration:none;color:#717188;">all our events</a>.
+                    </div>
+                    % endif
+                </td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+            ]]></field>
+    <field name="subject">${object.event_id.name}: ${object.get_date_range_str()}</field>
+    <field name="reply_to" eval="False"/>
+  </record>
+  <record id="event_mail_scheduler" model="ir.cron" forcecreate="True">
+    <field name="active"/>
```

Comment:
Can you remove this field? It is empty because it was removed. Just as well not to override it with a truthy value because if people disabled this task they probably won't want to have it re-enabled upon migration.